### DISCUSSION TITLE
README code example CI

### DISCUSSION THREAD

**Discussion by fnimick** [selfrefactor/rambda#746] [Ideas]
A thought: would it be worthwhile to (as a CI task) scan the README for code examples, and make sure they run without errors? We can easily get the body of the example by decoding the URL.

Alternatively, should we generate the examples from code (or tests) that can be run and confirmed working, and then insert them into the README as a build step?

This will solve a few instances I and others have found where examples do not work.

**Comment by selfrefactor** [comment]
testing can be useful as there were several cases where there were errors. I'd prefer to create e2e test for REPL site as the end result will be the same. I'll add it as a task; good suggestion.
